src/Features/Core/Portable/Completion/Providers/ImportCompletionProvider/ImportCompletionItem.cs

```diff
@@ -87,6 +88,9 @@ public static CompletionItem CreateAttributeItemWithoutSuffix(CompletionItem att
                  displayTextPrefix: attributeItem.DisplayTextPrefix,
                  displayTextSuffix: attributeItem.DisplayTextSuffix,
                  inlineDescription: attributeItem.InlineDescription);
+
+            item.Flags = flags;
+            return item;
```

Comment:
super bizarre that we can't pass this in through the constructor.

Reply:
I was trying to avoid (yet another) change to public API, right now `Flags` is internal.

Reply:
@genlu What would it take to make an internal Create method that passes it through? What's pretty terrifying here too is this field is *mutable* in what really should be an immutable type...

(This is fine if we have to do a mop-up, but let's make sure we actually do the mop-up!)

Reply:
Well, this is not the first mutable data on this type... But I totally agree, will do the mop-up in master after this is merged.